src/constants.telemetry.ts

```diff
@@ -230,6 +229,9 @@ export type TelemetryEvents = {
 		enabled: boolean;
 		version: string;
 	};
+	/** Temporary - sent when the user chooses to create a branch from the home view */
+	/** TODO: Replace this once we have source on all git commands*/
+	'home/createBranch': void;
```

Comment:
I think it is good to keep this regardless of the git command plumbing

Reply:
Maybe even add a `home/startWork' event to make it easier to track? @justinrobots

Reply:
I'll go ahead and add this. We can remove later if it feels redundant.